helpers/postgres.rb

```diff
@@ -60,7 +60,25 @@ def postgres_list
       }
     else
       dataset = dataset.eager(:representative_server, :timeline)
-      @postgres_databases = Serializers::Postgres.serialize(dataset.all, {include_path: true})
+      resources = dataset.all
+      # Group resources by their parent_id
+      grouped_resources = resources.group_by { |r| r[:parent_id] }
+
+      # Get all parents (resources with parent_id = nil)
+      parents = grouped_resources[nil] || []
+
+      # Create the ordered result
+      ordered_resources = []
+
+      # For each parent, add it followed by its children
+      parents.each do |parent|
```

Comment:
Maybe sort parents by their created_at as well? Also same for children, but in their own group?

Reply:
I believe the new code handles that